pylint/pyreverse/diagrams.py

```diff
@@ -121,16 +121,18 @@ def get_relationship(
     def get_attrs(self, node: nodes.ClassDef) -> list[str]:
         """Return visible attributes, possibly with class name."""
         attrs = []
-        properties = [
-            (n, m)
-            for n, m in node.items()
-            if isinstance(m, nodes.FunctionDef) and decorated_with_property(m)
-        ]
-        for node_name, associated_nodes in (
-            list(node.instance_attrs_type.items())
-            + list(node.locals_type.items())
-            + properties
+        properties = {
+            name: attr
+            for name, attr in node.items()
```

Comment:
I had to look into the code as well, but as I see it:

Here, `node` is of type `nodes.ClassDef`, and `node.items()` is a mapping from local name to the node defining the local:
https://github.com/pylint-dev/astroid/blob/600229f5dbe29ce9074b3c76a9d195951cb561a4/astroid/nodes/scoped_nodes/mixin.py#L179-L180

Thus `for local_name, local_node in node.items()` would be best IMO.